envi/archs/arm/disasm.py

```diff
@@ -1837,7 +1843,7 @@ def getOperAddr(self, op, emu=None):
         # if U==0, subtract
         addval *= pom
 
-        addr = (Rn + addval) & e_bits.u_maxes[self.tsize]
+        addr = Rn + addval
```

Comment:
please put this back as you found it.  i believe you accidentally made this change.